client/src/components/Tool/ToolForm.vue

```diff
@@ -284,7 +285,7 @@ export default {
                 this.onUpdate();
             }
         },
-        onUpdate() {
+        onUpdate: debounce(function (e) {
```

Comment:
This doesn't look quite right as a fix. If we've got those datasets in the initial form build we really don't need to fire onUpdate in the first place.

Reply:
It is hard to track where the change is coming from. So many components firing change events are involved here, that is the best solution I could come up with without having intimate knowledge about the tool form and tool build process :disappointed:

Maybe @guerler or someone else more familiar with the tool form can have a closer look?

Reply:
Thanks for looking into this. There was a `LastQueue` mechanism specifically built to avoid multiple redundant requests to the API. I would consider a regression in that mechanism to be a bug, it could prevent users from effectively using certain tools due to a large number of possibly slow requests.

Reply:
This is a workaround though, not a fix. If we come in with a defined set of inputs, there's no reason to fire off any requests.